test/cbmc/proofs/SigV4_GenerateHTTPAuthorization/SigV4_GenerateHTTPAuthorization_harness.c

```diff
@@ -75,6 +75,7 @@ void harness()
         __CPROVER_assume( pHttpParams->pathLen < MAX_URI_LEN );
         __CPROVER_assume( pHttpParams->queryLen < MAX_QUERY_LEN );
         __CPROVER_assume( pHttpParams->headersLen < MAX_HEADERS_LEN );
+        __CPROVER_assume( pHttpParams->pPayload != NULL );
```

Comment:
The payload can be NULL, no? If it's NULL, then it just uses the empty string.

Reply:
Has it been handled in the code that if payload is NULL then we should substitute it with empty string?

Reply:
I tested it locally before the release and the library should handle it